app/forms/news_article_form.rb

```diff
@@ -1,5 +1,14 @@
 class NewsArticleForm < NodeForm
   feature Reform::Form::MultiParameterAttributes
+  property :name, presence: true
```

Comment:
NewsArticleForm should inherit name from NodeForm. NodeForm doesn't validate the presence of name but it probably should anyway.